backend/src/types.ts

```diff
@@ -267,19 +267,30 @@ export interface INEUPrereqCourse {
  * @param scheduled The scheduled courses.
  */
 export interface ISchedule {
-    completed: ICompleteCourse[];
-    scheduled: string[][];
-    // todo: scheduled: IScheduleCourse[][];
```

Comment:
I think I mentioned to you at the last last meeting (not 10/20, but the one before) that this was the old Schedule data-definition, or at least I hope I did. In a more recent pull request, I moved the data definitions around so that the ones we're using are at the top, now merged into master. Message me on slack if you have any questions! I'm happy to help.